scripts/integration_test.py

```diff
@@ -808,6 +828,7 @@ def main():
 	import argparse
 
 	parser = argparse.ArgumentParser()
+	parser.add_argument("--show-full-output", action="store_true", help="print the full stdout and stderr on test failures")
 	parser.add_argument("--keep-tmpdirs", action="store_true", help="keep temporary directories used for the tests")
```

Comment:
```suggestion
	parser.add_argument("--keep-tmpdirs", action="store_true", help="keep temporary directories used for the tests")
	parser.add_argument("--show-full-output", action="store_true", help="print the full stdout and stderr on test failures")
```
Sort alphabetically, please.

Reply:
Done.